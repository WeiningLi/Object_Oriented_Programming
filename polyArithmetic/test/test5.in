create a 3/5 5 -2/5 2 1/2 1 3/7 0
print a 
create b 3/5 5 -2/5 2 1/2 1 3/7 0
print b
create c 3/5 6 -2/5 3 1/2 2 3/7 1
print c
subtract a b
add a b
add a c
add c a
subtract a c
subtract c a
multiply a a
multiply b b
multiply c c
multiply a b
multiply a c
divide c a
divide a c
move_copy a b
print b
