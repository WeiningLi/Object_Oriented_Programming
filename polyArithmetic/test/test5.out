(3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(3/5)x^6 + (-2/5)x^3 + (1/2)x^2 + (3/7)x
0
(6/5)x^5 + (-4/5)x^2 + (1)x + (6/7)
(3/5)x^6 + (3/5)x^5 + (-2/5)x^3 + (1/10)x^2 + (13/14)x + (3/7)
(3/5)x^6 + (3/5)x^5 + (-2/5)x^3 + (1/10)x^2 + (13/14)x + (3/7)
(-3/5)x^6 + (3/5)x^5 + (2/5)x^3 + (-9/10)x^2 + (1/14)x + (3/7)
(3/5)x^6 + (-3/5)x^5 + (-2/5)x^3 + (9/10)x^2 + (-1/14)x + (-3/7)
(9/25)x^10 + (-12/25)x^7 + (3/5)x^6 + (18/35)x^5 + (4/25)x^4 + (-2/5)x^3 + (-13/140)x^2 + (3/7)x + (9/49)
(9/25)x^10 + (-12/25)x^7 + (3/5)x^6 + (18/35)x^5 + (4/25)x^4 + (-2/5)x^3 + (-13/140)x^2 + (3/7)x + (9/49)
(9/25)x^12 + (-12/25)x^9 + (3/5)x^8 + (18/35)x^7 + (4/25)x^6 + (-2/5)x^5 + (-13/140)x^4 + (3/7)x^3 + (9/49)x^2
(9/25)x^10 + (-12/25)x^7 + (3/5)x^6 + (18/35)x^5 + (4/25)x^4 + (-2/5)x^3 + (-13/140)x^2 + (3/7)x + (9/49)
(9/25)x^11 + (-12/25)x^8 + (3/5)x^7 + (18/35)x^6 + (4/25)x^5 + (-2/5)x^4 + (-13/140)x^3 + (3/7)x^2 + (9/49)x
Quotient: (1)x
Remainder: 0
Quotient: 0
Remainder: (3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
(3/5)x^5 + (-2/5)x^2 + (1/2)x + (3/7)
